#!/usr/bin/env python
# field of view configuration

PACKAGE="pointcloud_preparation"

from dynamic_reconfigure.parameter_generator_catkin import *


gen = ParameterGenerator()

gen.add("dist_thresh", double_t, 1, "Outlier distance threshold", 0.01, 0.002, 0.1)
gen.add("max_iterations", double_t, 1, "Max Iterations", 50, 1, 200)

exit(gen.generate(PACKAGE, "pointcloud_preparation", "plane_segmentation_node"))
